growth/too/tasks/ref.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import subprocess
+import shlex
+from subprocess import PIPE
+import numpy as np
+import tempfile
+import requests
+
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from ..flask import app
+from . import celery
+from .. import models
+
+log = get_task_logger(__name__)
+
+__all__ = ('ztf_references',)
+
+
+@app.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_references():
+
+    user = app.config['IRSA_USERNAME']
+    pw = app.config['IRSA_PASSWORD']
+
+    if not 'IRSA_USERNAME' in app.config or not 'IRSA_PASSWORD' in app.config:
+        log.info('Missing IRSA_USERNAME or IRSA_PASSWORD...')
+        return
+
+    params = (('josso_cmd', 'login'),
+              ('josso_username', app.config['IRSA_USERNAME']),
+              ('josso_password', app.config['IRSA_PASSWORD']))
+
+    session = requests.Session()
```

Comment:
Please factor the creation of the TAP client session into a separate function, so that Tomas can reuse it:

```python
from pyvo.dal import TAPService

...

def get_tap_client():
    url = 'https://irsa.ipac.caltech.edu/TAP'
    return TAPService(url)
```